tasks/request_status_for_job/test/unit_tests/test_request_status_for_job_unit.py

```diff
@@ -0,0 +1,213 @@
+    ):
+        """
+        Basic path with all information present.
+        """
+        job_id = uuid.uuid4().__str__()
+        db_connect_info = Mock()
+
+        result = request_status_for_job.task(job_id, db_connect_info)
+
+        mock_get_granule_status_entries_for_job.assert_called_once_with(job_id, db_connect_info)
+        mock_get_status_totals_for_job.assert_called_once_with(job_id, db_connect_info)
+
+        self.assertEqual({
+            request_status_for_job.OUTPUT_JOB_ID_KEY: job_id,
+            request_status_for_job.OUTPUT_JOB_STATUS_TOTALS_KEY: mock_get_status_totals_for_job.return_value,
+            request_status_for_job.OUTPUT_GRANULES_KEY: mock_get_granule_status_entries_for_job.return_value
+        }, result)
+
+    @patch('database.result_to_json')
+    @patch('database.single_query')
+    def test_get_granule_status_entries_for_job_happy_path(
+            self,
+            mock_single_query: MagicMock,
+            mock_result_to_json: MagicMock
+    ):
+        """
+        Basic path with all information present.
+        """
+        job_id = uuid.uuid4().__str__()
+
+        db_connect_info = Mock()
+
+        result = request_status_for_job.get_granule_status_entries_for_job(job_id, db_connect_info)
+
+        mock_single_query.assert_called_once_with(f"""
+            SELECT
+                granule_id as \"{request_status_for_job.OUTPUT_GRANULE_ID_KEY}\",
+                orca_status.value AS \"{request_status_for_job.OUTPUT_STATUS_KEY}\"
+            FROM
+                orca_recoveryjob
+            JOIN orca_status ON orca_recoveryjob.status_id=orca_status.id
+            WHERE
+                job_id = %s
+            """,
+                                                  db_connect_info, (job_id,))
+        mock_result_to_json.assert_called_once_with(mock_single_query.return_value)
+
+        self.assertEqual(mock_result_to_json.return_value, result)
+
+    @patch('database.single_query')
+    def test_get_granule_status_entries_for_job_error_wrapped_in_database_error(
+            self,
+            mock_single_query: MagicMock
+    ):
+        """
+        If error is raised when contacting DB, error should be raised as a DatabaseError.
+        """
+        job_id = uuid.uuid4().__str__()
+        db_connect_info = Mock()
+
+        mock_single_query.side_effect = database.DbError()
+
+        with self.assertRaises(request_status_for_job.DatabaseError):
+            request_status_for_job.get_granule_status_entries_for_job(job_id, db_connect_info)
+
+    @patch('database.single_query')
+    def test_get_status_totals_for_job_happy_path(
+            self,
+            mock_single_query: MagicMock
+    ):
+        """
+        Basic path with all information present.
+        """
+        job_id = uuid.uuid4().__str__()
+        db_connect_info = Mock()
+
+        mock_single_query.return_value = [{'value': 'status0', 'total': 5}, {'value': 'status1', 'total': 10}]
```

Comment:
A little nit picky but it would be good if your return value was closer to what we would expect with 'pending', 'staged', 'success', 'failure' for values.

Reply:
I put one in, but I'd like this test to double-check that we aren't locked down to the pre-defined status values.